lib/src/utilities/dart_type_utilities.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.utilities.dart_type_utilities;
+
+import 'package:analyzer/dart/element/type.dart';
+
+class DartTypeUtilities {
+  static List<InterfaceType> findImplementedInterfaces(InterfaceType type,
```

Comment:
Something like this? https://github.com/alexeieleusis/sdk/commit/fbab68164d2d10b1fc06125b7d7aab4acff02db6

I didn't find a way to add a test for such a change, I guess contributing to that repo would require a more comprehensive documentation and testing/

I haven't worked yet with an SDK from a local git repository, hopefully is not that hard, would that require me to build only the analyzer?